src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/generated.py

```diff
@@ -0,0 +1,21 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+from __future__ import print_function
+
+from azure.cli.commands import cli_command, LongRunningOperation
+from ._factory import (iot_hub_service_factory,)
+from azure.cli.command_modules.iot.custom import \
+    (iot_hub_create, iot_device_create)
+
+# iot hub commands
+factory = iot_hub_service_factory
+
+cli_command('iot hub create', iot_hub_create, factory,
+            transform=LongRunningOperation('creating IoT Hub...', '', 10000.0))
```

Comment:
>transform=LongRunningOperation('creating IoT Hub...', '', 10000.0) [](start = 12, length = 66)

If you are fine with the default message (which would be 'Starting iot hub create') and the default polling interval (1 sec) then you could omit this. If 10 sec is more appropriate then that's fine--just want to make it clear that you don't have to specify this to make your long running operation work. :)